src/Package/Impl/Feedback/SendMailFallbackWindow.xaml

```diff
@@ -0,0 +1,29 @@
+﻿<!--
+    Copyright (c) Microsoft Corporation. All rights reserved.
+    Licensed under the MIT License. See LICENSE in the project root for license information.
+-->
+<Window x:Class="Microsoft.VisualStudio.R.Package.Feedback.SendMailFallbackWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Microsoft.VisualStudio.R.Package.Feedback"
+        mc:Ignorable="d"
+        Title="Send RTVS Feedback" Height="400" Width="600" WindowStartupLocation="CenterOwner">
+    <DockPanel LastChildFill="True">
+        <TextBlock x:Uid="Header" DockPanel.Dock="Top" Margin="10,10,10,5" TextWrapping="Wrap">
```

Comment:
@jflam, can you please take a look at the wording below and see if there's anything you'd like to change?